chatterbot/storage/__init__.py

```diff
@@ -2,3 +2,9 @@
 from .django_storage import DjangoStorageAdapter
 from .jsonfile import JsonFileStorageAdapter
 from .mongodb import MongoDatabaseAdapter
+
+# FIXME Better way manage import
+try:
```

Comment:
I think these files are static and reside in same directory as long as they are. It's always a safe import.